examples/data-objects/table-document/package.json

```diff
@@ -29,12 +29,12 @@
 		"prettier": "prettier --check . --ignore-path ../../../.prettierignore",
 		"prettier:fix": "prettier --write . --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
-		"test:coverage": "nyc npm test -- --reporter xunit --reporter-option output=nyc/junit-report.xml --exit",
+		"test:coverage": "c8 npm test",
```

Comment:
Just FYI - I noticed that some of the 'test:coverage' scripts passed an '--exit' argument.  I assume this isn't required, since we've been invoking coverage from the root with c8.

Reply:
Also, I already enforced that the repo use the common config, which has --exit on by default.